### PR TITLE
add daemonsets for prepulling images

### PR DESCRIPTION
**Pull request by colearendt**
this can also be used to "force a repull" of images that have been cached on the cluster. Say... in the case that you need a security patch 😅

### CODE REVIEW

**Review by jeffvroom — APPROVED** [review]
LGTM!

**Review by MariaSemple — APPROVED** [review]
LGTM!

File: examples/yaml/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+They differ only in which repository image they re-pull. This differs based on:
+
+- Repository: `ghcr.io` (GitHub container registry) or `DockerHub`.
+    - `DockerHub` images look like `rstudio/r-session-complete:2021.09.2-382.pro1`
+    - `ghcr.io` images look like `ghcr.io/rstudio/r-session-complete:2021.09.2-382.pro1`
+- Product version
+
+Feel free to download one of these files and modify it for your purposes. It is important that your
+daemonset references the exact same images that are used by your RStudio Workbench installation.
```
Comment (jeffvroom) [inline examples/yaml/README.md:18]:
```suggestion
daemonset references the exact same images that are used by your RStudio Workbench installation to ensure the image you are using is updated.  
```

File: examples/yaml/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+### What is it
+
+A
+[daemonset](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)
+runs a single pod on every node of a Kubernetes cluster. In this case, we
+deploy a daemonset that has an `initContainer` with `imagePullPolicy: "Always"`
+that runs a simple `echo` command. Then the `pause` container takes over, and
+nothing else happens for the life of the daemonset.
```
Comment (jeffvroom) [inline examples/yaml/README.md:27]:
```suggestion
nothing else happens for the life of the daemonset. When run in your kubernetes environment, this daemonset will force the specified image to be updated on each node. Later when sessions are started on that same node, they will use the updated image. 
```

**Review by jeffvroom — APPROVED** [review]
Looks good. Just made a couple suggestions to add a little more context.

**Pull request merged**